### PR TITLE
fix #701 properties.urlQuotes doesn't work

### LINKED ISSUE #701: `properties.urlQuotes` doesn't work now
https://github.com/jakubpawlowicz/clean-css/blob/master/lib/urls/rewrite.js#L95 remove all possible quotes.

### PR DESCRIPTION
**Pull request by island205**
Prevent SourceReader to remove all quote in `url()`

### CODE REVIEW

**Comment by jakubpawlowicz** [comment]
Could you please add a spec covering this change, too? It should go into `test/integration-test.js`. I have trouble seeing how this change affects the `properties.urlQuotes` without (at least) one.

**Comment by island205** [comment]
@jakubpawlowicz test case added.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 /cleancss-browser.js
 /node_modules
 /npm-debug.log
+reset.css
```
Comment (jakubpawlowicz) [inline .gitignore]: This bit is not necessary.

**Comment by jakubpawlowicz** [comment]
And please squash commits into one. Thanks!

**Comment by island205** [comment]
@jakubpawlowicz updated!

**Comment by jakubpawlowicz** [comment]
Much appreciated @island205, will land as a fix in 3.4.7.

**Pull request merged**